proto/tpm_enrollz.proto

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" B1IS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+syntax = "proto3";
+
+package openconfig.attestz.proto;
+
+import "github.com/openconfig/attestz/proto/common_definitions.proto";
+
+option go_package = "github.com/openconfig/attestz";
+
+message GetIakCertRequest {
+  // Switch control card selected identifier.
+  ControlCardSelection control_card_selection = 1;
+}
+
+message GetIakCertResponse {
+  // Vendor identity fields of the selected control card.
+  ControlCardVendorId control_card_id = 1;
+
+  // PEM-encoded initial attestation key certificate signed by the switch
+  // vendor CA.
+  string iak_cert = 2;
+
+  // PEM-encoded initial DevID certificate signed by the switch vendor CA.
+  // This field should only be populated for standby/secondary control card.
+  string idevid_cert = 3;
+}
+
+message StoreOIakCertRequest {
```

Comment:
yes, we would call this API once for each control card. I think `Store` is a better name because it applies to both "install" and "rotate" use cases.

Reply:
i don't think that verb implies that store to me === Put which then says there can be any number of these

Rotate always implies there can only be one of these and if it doesn't exist before rotating None to something is still a fine verb

Lastly at least get clearer then definitions in the service decription that explain the api more completely

Reply:
Ok, that's fine, we can change it to `RotateOwnerAttestationCertficate`, and I will add description to every API in this PR.